modules/microshift-config-yaml.adoc

```diff
@@ -4,81 +4,43 @@
+= Configuring {product-title}
 
-Configure your {product-title} using a YAML file. The values specified in your configuration file are used when you execute a command.
-// Q: can these values be overriden with CLI flags?
+{product-title} searches for a configuration file in the user-specific directory, `~/.microshift/config.yaml`, then the system-wide `/etc/microshift/config.yaml` directory. Create the configuring file, specifying any settings that should override the defaults, before starting MicroShift.
 
 [id="microshift-yaml-default_{context}"]
 == Default settings
 If you do not create a `config.yaml` file, the default values are used.
-// Q: what creates the default yaml?
-// Q: can default file be modified? (instead of making a new one)
 
 .Default YAML file example
 [source,yaml]
 ----
-cluster:
-  clusterCIDR: 10.42.0.0/16
-  serviceCIDR: 10.43.0.0/16
-  serviceNodePortRange: 30000-32767
-  dns: 10.43.0.10
-  domain: cluster.local
-  url: https://127.0.0.1:6443
-nodeIP: ""
-nodeName: ""
-logVLevel: 0
+dns:
+  baseDomain: microshift.example.com <1>
+network:
+  clusterNetwork:
+    - cidr: 10.42.0.0/16 <2>
+  serviceNetwork:
+    - 10.43.0.0/16 <3>
+  serviceNodePortRange: 30000-32767 <4>
+node:
+  hostnameOverride: "" <5>
+  nodeIP: "" <6>
+apiServer:
+  subjectAltNames: [] <7>
+debugging:
+  logLevel: "Normal" <8>
 ----
-
-[IMPORTANT]
+<1> Base domain of the cluster. All managed DNS records will be sub-domains of this base.
+<2> A block of IP addresses from which Pod IP addresses are allocated.
+<3> A block of virtual IP addresses for Kubernetes services.
+<4> The port range allowed for Kubernetes services of type NodePort.
+<5> The name of the node, defaults to hostname.
+<6> The IP address of the node, defaults to IP of the default route.
+<7> Subject Alternative Names for apiserver certificates.
+<8> Log verbosity. Valid values are Normal, Debug, Trace, or TraceAll.
+
+[NOTE]
 ====
-The configuration file must be located at the user-specific directory, `~/.microshift/config.yaml`, and the system-wide `/etc/microshift/config.yaml` directory. An existing user-specific `config.yaml` takes precedence.
+MicroShift only reads the configuration file on startup. Restart MicroShift after changing any configuration settings to have them take effect.
 ====
-//Q: can this be done with a command, or is this an instance where user goes into the directory and creates a text file?
-
-[id="microshift-yaml-format-example_{context}"]
-== Formatting a customized YAML file
-
-.Procedure
-//Q: need explicit steps
-. Create a YAML file with the following formatting:
-+
-[source,yaml]
-----
-cluster: ""
-  clusterCIDR: ""
-  serviceCIDR: ""
-  serviceNodePortRange: ""
-  dns: ""
-  domain: ""
-  url: ""
-nodeIP: ""
-nodeName: ""
-logVLevel: ""
-----
-
-. Enter valid values.
-//Q: most docsets contain a table of valid entry types, noting whether required or optional
-//Q: does the user need to enter every value? will any default?
```

Comment:
All of them have defaults. Not all of the defaults are necessarily what someone would want, but they do work.